corehq/apps/hqwebapp/management/commands/migrate_app_to_bootstrap5.py

```diff
@@ -39,6 +42,12 @@ def add_arguments(self, parser):
             default=False,
             help="Run migration against already split bootstrap 5 files"
         )
+        parser.add_argument(
+            '--references',
```

Comment:
nit: this argument name seems like it would take a value rather than being a boolean flag. What do you think of renaming it to `--verify-references`?

If it will be used often it could be enabled by default and add an option to turn it off (`--no-verify-references`) or alternately, could add an additional single-letter shorthand option like `-r`.